src/views/Project/index.jsx

```diff
@@ -289,7 +290,11 @@ export default class Project extends Component {
           bugzillaData.bugs.edges.map(edge => edge.node).map(bug => ({
             assignee: bug.status === 'ASSIGNED' ? bug.assignedTo.name : '-',
             project: bug.component,
-            tags: bug.keywords || [],
+            tags: [
+              ...(bug.keywords || []),
+              ...((bug.whiteboard && extractWhiteboardTags(bug.whiteboard)) ||
```

Comment:
If you include a check in `extractWhiteBoardTags` that returns an empty array if `bug.whiteboard` is invalid, then you can do:

```jsx
tags: [
  ...(bug.keywords || []),
  ...(extractWhiteboardTags(bug.whiteboard)),
],

```